mParticle-Apple-SDK/Utils/NSNumber+MPFormatter.m

```diff
@@ -4,18 +4,15 @@ @implementation NSNumber(MPFormatter)
 
 - (NSNumber *)formatWithNonScientificNotation {
     double minThreshold = 1.0E-5;
-    double maxThreshold = 1.0E10;
     double selfAbsoluteValue = fabs([self doubleValue]);
     NSNumber *formattedNumber;
     
     if (selfAbsoluteValue < minThreshold) {
         formattedNumber = @0;
-    } else if (selfAbsoluteValue < maxThreshold) {
-        formattedNumber = self;
     } else {
         NSNumberFormatter *numberFormatter = [[NSNumberFormatter alloc] init];
-        numberFormatter.maximumFractionDigits = 5;
-        numberFormatter.numberStyle = NSNumberFormatterNoStyle;
+        numberFormatter.numberStyle = NSNumberFormatterDecimalStyle;
+        numberFormatter.maximumFractionDigits = 2;
         NSString *stringRepresentation = [numberFormatter stringFromNumber:self];
         formattedNumber = @([[numberFormatter numberFromString:stringRepresentation] doubleValue]);
```

Comment:
```suggestion
        formattedNumber = [numberFormatter numberFromString:stringRepresentation];
```
Looks like we shouldn't need that extra round trip from doubleValue back to NSNumber

Reply:
Addressed this change